### PR TITLE
Add io.github.subhra74.Muon

### PR DESCRIPTION
**Pull request by DUB1401**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

**Description:** This application is one of the most convenient SSH clients, and its author has already taken care of publishing it as a [Flatpak](https://github.com/subhra74/snowflake/tree/master/flatpak), but did not finish the job and did not publish the assembly in Flathub. I fix some AppStream and permissions issues.

### CODE REVIEW

**Comment by DUB1401** [comment]
bot, build io.github.subhra74.Muon

File: io.github.subhra74.Muon.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.openjdk11"
+	],
+	"command": "snowflake",
+	"finish-args": [
+		"--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+		"--share=ipc",
+		"--socket=x11",
+		"--share=network",
+		"--filesystem=home"
```
Comment (bbhtt) [inline io.github.subhra74.Muon.json:15]: Why do you need home access?

Comment (DUB1401) [reply io.github.subhra74.Muon.json:15]: Because this app also supports SFTP, file transferring often provides cover home directory.

**Comment by bbhtt** [comment]
Would this require `--socket=ssh` ?

File: io.github.subhra74.Muon.json (hunk 1 above)
Comment (bbhtt) [reply io.github.subhra74.Muon.json:15]: Can't you make it specific a single directory like xdg-downloads or xdg-documents?

Comment (DUB1401) [reply io.github.subhra74.Muon.json:15]: This is not even very convenient for me personally, and some of the functions of the native application are cut off. Then it makes no sense for me to publish it in Flatpak.

**Comment by DUB1401** [comment]
bot, build io.github.subhra74.Muon

**Pull request closed without merging**